nextflow.config

```diff
@@ -61,7 +58,7 @@ params {
 
     // GAS Cluster
     gm_thresholds = "10,5,0"
-    gm_delimiter = "'.'" // note the single quotes surrounding the delimiter
+    gm_delimiter = "."
```

Comment:
The `nf-core lint` linter does not seem to properly handle double-quotes surrounded by single-quotes and was causing issues.

Instead of requiring the parameter to include quotes around it, I added the quotes to the `gas mcluster` command in the respective process.